tools/scripts/generic.mk

```diff
@@ -0,0 +1,13 @@
+ifeq (aducm3029,$(strip $(PLATFORM)))
```

Comment:
I think that what this file does is something that we all want so just make sure in final version (not this draft) to combine these ifs to one single if - else if.. statement, now it's unnecessarily complicated.

Reply:
aducm.mk works both for windows and linux. Meanwhile for altera and xilinx you need to choose one .mk depending on the os, I don't know how to simplify this if without making the other makefiles generic too.